invenio_theme/webpack.py

```diff
@@ -71,7 +71,7 @@
                 '@js/invenio_theme': 'js/invenio_theme',
                 # TODO: change to "@less/invenio_theme":
```

Comment:
I think this PR fixes this TODO, so it can be removed.

Reply:
Indeed, thanks for the catch

Reply:
well... fixed after merge, I gess for the next time 😓